src/MCPClient/tests/test_move_or_merge.py

```diff
@@ -1,5 +1,6 @@
 # -*- encoding: utf-8
 
+from os import listdir
```

Comment:
I think you can use pytest for this:

https://docs.pytest.org/en/latest/tmpdir.html#the-tmpdir-fixture
https://py.readthedocs.io/en/latest/path.html#py._path.local.LocalPath.listdir

Reply:
That's a nice tip, thank you!